.github/workflows/CI.yml

```diff
@@ -13,9 +12,10 @@ jobs:
         os: [windows-latest, ubuntu-latest]
     runs-on: ${{matrix.os}}
     outputs: # https://stackoverflow.com/questions/59175332/using-output-from-a-previous-job-in-a-new-one-in-a-github-action
-      Version: ${{ steps.gitversion.outputs.nuGetVersionV2 }}
+      semVer: ${{ steps.gitversion.outputs.semVer }}
```

Comment:
Why? They look almost identical. What is the advantage? 

"SemVer": "0.18.2-PullRequest0298.8",
"NuGetVersionV2": "0.18.2-pullrequest0298-0008",

Reply:
A bit of personal preference, but also semantic version is cross-platform terminology.
i.e. a 100% Node/Python developer doesn't know a thing about NuGet... but they know what a semver is.